javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -30,7 +30,11 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
 1. Create a bare-bones homepage for a restaurant. Include an image, headline, and some copy about how wonderful the restaurant is. It’s okay to hard-code these into the HTML for now just to see how they look on the page.
 
 
-1. Now remove those elements from the HTML (so leave only the `<html>`, `<body>`, and `<div id="content">` tags) and instead create them by using JavaScript only, e.g. by appending each new element to `div#content` once the page is first loaded. Since we're all set up to write our code in multiple files, let's write this initial page-load function inside of its own module and then import and call it inside of `index.js`.
+1. Now remove those elements from the HTML and instead create them by using JavaScript only, e.g. by appending each new element to the body once the page is first loaded. 
```

Comment:
I might just not be understanding what you're intending, but why do we need learners to delete individual elements from their HTML file if in the next step they'll be deleting it?